### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by cdflynn**
I have an empty `HELP.md`, and my intention is to link to that from the main readme, with something like

```
Not familiar with these (patterns/features/libraries... w/e?), check out the [help document link].  You'll find some concise explanations and links to more in-depth explanations.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Zeta Bootstrap (Work In Pogress)
```
Comment (manjunathc23) [inline README.md:1]: :)

File: README.md [hunk 2]
```diff
@@ -13,116 +13,70 @@
...
+Zeta comes with these (and more) pre-configured, so you can start writing _your_ code, _now_.
 
-`#Testing Matters (TDD)` 
+With Zeta there's no reason to do without common good practices, even in your prototypes.
 
-####What needs to done from your end to start with? 
+###What do I need to do to start using Zeta?
 
-1. Just drop your fragments in place and follow some of the design patterns. That's it!
+In three words: **fork and go**.
```
Comment (manjunathc23) [inline README.md:39]: 👍

**Review by manjunathc23 — COMMENTED** [review]
Read me looks simple and neat so far.

**Comment by manjunathc23** [comment]
Rebase to latest `master` should fix the build error in here.

**Comment by cdflynn** [comment]
for some reason I had this branched from my package structure re-org instead of `master`.  I changed that.

**Pull request merged**